### DISCUSSION TITLE
Burmese font and Keycode keybindings

### DISCUSSION THREAD

**Discussion by minsoehan** [zed-industries/zed#22034] [Feature Requests]
I tested zed editor. It's great. But I have two problems. One is Myanmar/Burmese Font. I opened an issue here #21148 and the other is Keycode like --to-code option for bindsym in sway config file so I can do Ctrl+Shift+k to delete line while I am on Non-English keyboard layout.
The two above is important, I think, many people including me are using a text editor to write academic writing in their own language directly in HTML file. For instance, I want an advance text editor in which I can type Non-english fonts and with LSP for HTML, CSS, JS and so on, and fast and small. I'm not a coder but a writer.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
